zetaclient/chains/evm/observer/v2_tracker.go

```diff
@@ -0,0 +1,88 @@
+package observer
+
+import (
+	"context"
+	"fmt"
+
+	ethtypes "github.com/ethereum/go-ethereum/core/types"
+	"github.com/onrik/ethrpc"
+	"github.com/zeta-chain/protocol-contracts/v2/pkg/gatewayevm.sol"
+
+	"github.com/zeta-chain/node/zetaclient/zetacore"
+)
+
+// ProcessInboundTrackerV2 processes inbound tracker events from the gateway
+func (ob *Observer) ProcessInboundTrackerV2(
```

Comment:
Can you add a unit test for this?

Reply:
I think coverage requires some extra effort. The unit tests may come later as long as the E2E tests passed.

Reply:
It would be some efforts for tests that would then still be refactored afterward since we need a proper test suite for the chain client